docker/Dockerfile.centos.7

```diff
@@ -81,8 +81,8 @@ RUN yum -y copr enable jhli/safeclib
 RUN yum -y  install libipmctl-devel
 
 RUN yum -y install librdmacm-devel valgrind valgrind-devel libibverbs-devel \
-           python-jwt python-virtualenv python2-pip python34-pip            \
-           python34-pygithub
+           python-jwt python-virtualenv python2-pip python36 python36-pip   \
```

Comment:
```suggestion
           python-jwt python-virtualenv python2-pip python36-pip   \
```

Reply:
I can't see anything here that uses either pip or pip3.  I also note that this Dockerfile pulls in scons (the non-python3 version) and pylint

Reply:
Isn't pip3 python3-pip?   Anyway, I can't change the scons version without cart and daos supporting a python3 build because the scons_local testing tries to build daos and cart.